content/connector.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2019-07-31
+slug: about-connector
+type: post
+title: NWISWeb Tableau Data Connector
+categories:
+  - Web Data Connector
+description: Information about the NWISWeb Tableau Data Connector.
+keywords:
+  - Tableau
+  - Web
+  - Data
+  - Connector
+---
+
+# NWISWeb Tableau Web Data Connector
+
+- How to use the NWISWeb Tableau Web Data Connector
+This system requires a desktop Tableau client. To use this system, first install a Tableau desktop client on your PC then enter the URL for this page under the web data connector option in the connect menu. Details of the constraints imposed by the database that feeds this tool are available here.
```

Comment:
Sorry, one more thing.  You probably want to put the url of the actual connector in (I think we settled on labs.waterdata.usgs.gov/tableau-connector)

Reply:
sounds good, I'll add that.